enumerable.rb

```diff
@@ -0,0 +1,26 @@
+require_relative './my_enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*args)
+    @list = args
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] MyList class should include an `#each` method that yields all members of `@list` in succession, kindly implement this method on your own, you should not use the `.each` ruby method